system/core/events.php

```diff
@@ -53,12 +53,6 @@ function addEventHandler($html) {
 		ob_start();
 ?>
 <script>
-	/*
-		Variable: baseUrl
-		Javascript variable containing the base url from the document location object.
-	*/
-	var postProcessingList = new Array();
```

Comment:
This removes the `postProcessingList` without comment. Is it unused and therefore removed? Is it replaced by something else? This should probably be a separate commit in any case?

Reply:
I've moved this, and the addition of the callback parameter to `hypha()`, into seperate commits.